lib/routes/account.js

```diff
@@ -484,13 +516,22 @@ module.exports = function (
                   uid: emailRecord.uid,
                   kA: emailRecord.kA,
                   wrapKb: wrapKb,
-                  emailVerified: emailRecord.emailVerified
+                  emailVerified: emailRecord.emailVerified,
+                  tokenVerificationId: tokenVerificationId
                 })
               }
             )
             .then(
               function (keyFetchToken) {
                 response.keyFetchToken = keyFetchToken.data.toString('hex')
+                response.challengeMethod = 'email'
+
+                if(! emailRecord.emailVerified) {
+                  response.challengeReason = 'signup'
+                } else {
+                  response.challengeReason = 'signin'
+                }
+
                 return response
```

Comment:
`verified: false` needs to be set on the response or else existing clients won't behave correctly.